### ISSUE TITLE
showPath option is not doing anything

### ISSUE THREAD

**Issue by lookapanda** [thebigredgeek/apollo-errors#27]
> showPath (false): Preserve the GraphQLError path data.

```javascript
createError('SomeError', {
    message: 'Some error message.',
    options: {
        showLocations: false,
        showPath: true,
    },
});
```

**Expected Behaviour**
It shows the path in the GraphQL result.

**Actual Behaviour**
It is not showing the path, only if I enable `showLocations` too, which I don't want.

**Comment by thebigredgeek** [comment]
Can you submit a PR?

**Comment by thebigredgeek** [comment]
Should be fixed in [1.7.0](https://github.com/thebigredgeek/apollo-errors/releases/tag/1.7.0)
